CHANGES.rst

```diff
@@ -20,6 +20,8 @@ Unreleased
   and ``welcome_message_text``. This change also removes the need for PRAW
   to track current subreddit settings and send unmodified ones in the
   update request.
+* :class:`.SubredditRules` now correctly returns the unobjectified dict when
+  called.
```

Comment:
Let's not list this in the changelog, or at least not list it this way. From the perspective of a PRAW user using the existing behavior, from 7.0.0 to whatever this becomes (7.1.0?), nothing should have changed other than their existing workflow becoming deprecated, which we should list in the changelog.

Reply:
Oops, @bboe beat me to it in making this comment.